cmd/cli/kubectl-kyverno/commands/apply/command_test.go

```diff
@@ -77,9 +77,10 @@ func Test_Apply(t *testing.T) {
 		},
 		{
 			config: ApplyCommandConfig{
-				PolicyPaths:   []string{"../../../../../test/cli/apply/policies"},
-				ResourcePaths: []string{"../../../../../test/cli/apply/resource"},
-				PolicyReport:  true,
+				PolicyPaths:      []string{"../../../../../test/cli/apply/policies"},
+				ResourcePaths:    []string{"../../../../../test/cli/apply/resource"},
+				PolicyReport:     true,
+				policyExceptions: true,
```

Comment:
Instead of modifying an existing unit test, I’d suggest adding three new ones:

1. A test that loads exceptions from the policy path. It should create three resources: one that matches the exception, and two that match the policy, one of which should fail and the other pass.
2. A test that loads exceptions from the resource path, again creating three resources following the same logic.
3. A test that loads exceptions from both the policy and resource paths.